pkg/kubectl/configmap.go

```diff
@@ -90,8 +85,26 @@ func (s ConfigMapGeneratorV1) Generate(genericParams map[string]interface{}) (ru
 		delegate.EnvFileSource = fromEnvFile
 		delete(genericParams, "from-env-file")
 	}
+	hashParam, found := genericParams["hash"]
+	if found {
+		hashBool, isBool := hashParam.(bool)
```

Comment:
Very minor nit: I think you can assign `delegate.Hash` directly instead of using the temporary `hashBool`.

Also applies to secret code below.